docs/schema/guidance/information-updates.rst

```diff
@@ -34,4 +34,22 @@ Once a series of Statements about a given element is closed, further Statements
 
 When the subject or the interested party of a relationship change, the lifecycle of that relationship is considered to have ended. For example, if the owner of a company transfers their interest to a relative, the ``recordStatus`` of the first relationship would be 'closed', and - potentially - details of a new record would be published.
 
+Requirements for special cases
+------------------------------
 
+
+Error correction
+++++++++++++++++
+
+Errors in published data may be due to mistakes at the point of information disclosure, or to the mis-processing of information by the data management system. In either case, errors SHOULD be corrected by the issuing of new statements:
```

Comment:
not sure about mis-processing - it potentially translates a bit vaguely. Maybe 'incorrect processing' ?

Reply:
Yes. Will edit.